arrows/ffmpeg/tests/test_video_input_ffmpeg.cxx

```diff
@@ -52,86 +68,75 @@ TEST_F ( ffmpeg_video_input, create )
+
+  // Close the video
   input.close();
-  EXPECT_FALSE( input.good() ) <<
-    "Video state after close";
+  EXPECT_FALSE( input.good() ) << "Video state after close";
 }
 
 // ----------------------------------------------------------------------------
 TEST_F ( ffmpeg_video_input, is_good_invalid_file_path )
 {
-  kwiver::arrows::ffmpeg::ffmpeg_video_input input;
-  kwiver::vital::path_t incorrect_file = data_dir + "/DoesNOTExists.mp4";
-
-  EXPECT_FALSE( input.good() ) <<
-    "Video state before open";
+  ffmpeg::ffmpeg_video_input input;
+  auto const filename = data_dir + "/DoesNOTExist.mp4";
 
-  // open the video
-  EXPECT_THROW(
-    input.open( incorrect_file ),
-    kwiver::vital::file_not_found_exception );
-  EXPECT_FALSE( input.good() ) <<
-    "Video state after open but before first frame";
+  // Open the video
+  EXPECT_THROW( input.open( filename ), kv::file_not_found_exception );
+  EXPECT_FALSE( input.good() )
+    << "Video state after open but before first frame";
 
   // Get the next frame
-  kwiver::vital::timestamp ts;
-  EXPECT_THROW( input.next_frame( ts ),
-                kwiver::vital::file_not_read_exception );
-  EXPECT_EQ( ts.get_frame(), 0 ) << "Initial frame value mismastch";
-  EXPECT_FALSE( input.good() ) <<
-    "Video state after open but before first frame";
-
-  // close the video
+  kv::timestamp ts;
+  EXPECT_THROW( input.next_frame( ts ), kv::file_not_read_exception );
+  EXPECT_EQ( ts.get_frame(), 0 ) << "Initial frame value mismatch";
+  EXPECT_FALSE( input.good() ) << "Video state after first frame";
+
+  // Close the video
   input.close();
-  EXPECT_FALSE( input.good() ) <<
-    "Video state after close";
+  EXPECT_FALSE( input.good() ) << "Video state after close";
 }
 
 // ----------------------------------------------------------------------------
 TEST_F ( ffmpeg_video_input, frame_image )
 {
-  kwiver::arrows::ffmpeg::ffmpeg_video_input input;
-  kwiver::vital::path_t correct_file = data_dir + "/" + video_name;
-
-  EXPECT_FALSE( input.good() ) <<
-    "Video state before open";
+  ffmpeg::ffmpeg_video_input input;
 
-  // open the video
-  input.open( correct_file );
-  EXPECT_FALSE( input.good() ) <<
-    "Video state after open but before first frame";
-  EXPECT_EQ( input.frame_image(),
-             nullptr ) << "Video should not have an image yet";
+  // Open the video
+  input.open( ffmpeg_video_path );
+  EXPECT_FALSE( input.good() )
+    << "Video state after open but before first frame";
+  EXPECT_EQ( input.frame_image(), nullptr )
+    << "Video should not have an image yet";
 
   // Get the next frame
-  kwiver::vital::timestamp ts;
+  kv::timestamp ts;
   input.next_frame( ts );
   EXPECT_EQ( ts.get_frame(), 1 );
 
-  kwiver::vital::image_container_sptr frame = input.frame_image();
+  auto const frame = input.frame_image();
   EXPECT_EQ( frame->depth(), 3 );
   EXPECT_EQ( frame->get_image().width(), 80 );
   EXPECT_EQ( frame->get_image().height(), 54 );
```

Comment:
I think the tests would benefit from the error output messages like some other ones have. Unless the output of these are verbose enough? (Generically a remark on the file, not just here)

Reply:
I'm not sure what further information is needed?